app/assets/javascripts/application.js

```diff
@@ -30,9 +30,14 @@ jQuery(function($) {
 
   var $form = $('.js-live-search-form'),
       $results = $('.js-live-search-results-block');
+      $atom_autodiscovery_link = $("link[type='application/atom+xml']").eq('0');
 
   if($form.length && $results.length){
-    new GOVUK.LiveSearch({$form:$form, $results:$results});
+    new GOVUK.LiveSearch({
+      $form:$form,
+      $results:$results,
+      $atom_autodiscovery_link:$atom_autodiscovery_link
```

Comment:
Sorry, pedantry ahoy:

    Use camelCase when naming objects, functions, and instances. 

https://github.com/alphagov/styleguides/blob/master/js.md#naming-conventions